.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,5 @@
+# No Template
+
+👉 Please follow one of the issue templates provided by the repo - if you are seeing this message, it means you haven't.
+
+If you don't follow the issue template, we may immediately close it.
```

Comment:
Where does this occur? Would this be an explicit option when opening a new issue?

Reply:
Developers would see this as a "slightly hidden" option in the selection, it's the last line here -> https://github.com/facebook/react-native/issues/new/choose